front/app/component-library/utils/styleUtils.ts

```diff
@@ -258,6 +258,17 @@ export const stylingConsts = {
   border: `1px solid ${colors.divider}`,
 };
 
+export const getFontWeightCSS = (fontWeight: FontWeight | undefined) => {
+  switch (fontWeight) {
+    case 'bold':
+      return 'bold';
+    case 'medium':
+      return 600;
+    default:
+      return 'normal';
+  }
+};
```

Comment:
font-weight: '600' wasn't valid CSS, so instead I went for a more descriptive solution which matches what we already have for the FontWeight type.

So now we can use 'normal', 'bold', or 'medium' in Text/Titles, and the medium is just transformed to 600 in the CSS later :)

Reply:
Okay, but let's still add the comments still on what "bold" and "normal" translate to (700 and 400)?